src/main/java/com/google/graphgeckos/dashboard/datatypes/Log.java

```diff
@@ -55,4 +63,22 @@ public String getLink() {
     return link;
   }
 
+  /**
+   * Sets the log type. 
+   *
+   * @param type the new value for {@code type}. Should not be null.
+   */
+  public void setType(String type) {
```

Comment:
Consider adding a `@NonNull` annotation on the type and link.

Reply:
Actually I've mistakenly copied the `Should not be null` part here. Both of these fields can be null. I updated the PR accordingly.